app/controllers/api/v1/translations_controller.rb

```diff
@@ -37,9 +37,12 @@ def create
     revert_resource_name_to_controller_type
 
     translation = Translation.transaction(requires_new: true) do
-      create_params[:translated_type] = params[:translated_type].classify
-      # TODO: should raise an error if this is missing
-      create_params[:translated_id] = params[:translated_id]
+
+      # push these scope params into create payload
+      # to validate using the create schema
+      params[:translations][:translated_type] = params[:translated_type].classify
```

Comment:
Are there any security risks with calling `classify`?

Reply:
I don't think so, it converts string to string e.g. `"project"` ->  `"Project"`. 
That is then fed into the create params for Rails to interpret as a valid polymorphic class target (i.e. does that class name exist)